src/universal/conf/cassandra-env.sh

```diff
@@ -96,4 +96,7 @@ fi
 # to the location of the native libraries.
 JVM_OPTS="$JVM_OPTS -Djava.library.path=$CASSANDRA_HOME/lib/sigar-bin"
 
-JVM_OPTS="$JVM_OPTS $JVM_EXTRA_OPTS"
\ No newline at end of file
+JVM_OPTS="$JVM_OPTS $JVM_EXTRA_OPTS"
+
+# ConductR: Replace dead node when restarting a new Cassandra instance on the same address
+JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address=$CAS_STORAGE_BIND_IP"
```

Comment:
Gosh, they're making life tricky. Perhaps startup without the option, and if that fails then try again with the option?